stock_request_partner/readme/USAGE.rst

```diff
@@ -0,0 +1,7 @@
+## Creation
+
+* Go to 'Stock Requests / Stock Requests' and create a new Request.
+* Indicate a partner, product, quantity and location.
+* Press 'Confirm'.
+
+In case that transfers are created, the partner will defined in the related transfers.
```

Comment:
@alan196  You maybe want to speak about procurement group auto creation if a partner is set ?

Reply:
@rousseldenis 👍 readme updated